sonar-python-plugin/src/main/java/org/sonar/plugins/python/pylint/PylintSensor.java

```diff
@@ -19,71 +19,70 @@
  */
 package org.sonar.plugins.python.pylint;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
 import org.apache.commons.io.FileUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.sonar.api.batch.Sensor;
-import org.sonar.api.batch.SensorContext;
 import org.sonar.api.batch.fs.FilePredicates;
 import org.sonar.api.batch.fs.FileSystem;
 import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.batch.rule.ActiveRule;
-import org.sonar.api.batch.rule.ActiveRules;
-import org.sonar.api.component.ResourcePerspectives;
+import org.sonar.api.batch.sensor.Sensor;
+import org.sonar.api.batch.sensor.SensorContext;
+import org.sonar.api.batch.sensor.SensorDescriptor;
+import org.sonar.api.batch.sensor.issue.NewIssue;
 import org.sonar.api.config.Settings;
-import org.sonar.api.issue.Issuable;
-import org.sonar.api.resources.Project;
 import org.sonar.api.rule.RuleKey;
 import org.sonar.plugins.python.Python;
 
-import java.io.File;
-import java.io.IOException;
-import java.util.List;
-
 public class PylintSensor implements Sensor {
-  public static final String REPORT_PATH_KEY = "sonar.python.pylint.reportPath";
 
   private static final Logger LOG = LoggerFactory.getLogger(PylintSensor.class);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "LOG" private field. ([squid:S1068](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-235b&issues=AWD6WlEaf6JiG_HArhdB&open=AWD6WlEaf6JiG_HArhdB)

[ISSUE_KEY]: AWD6WlEaf6JiG_HArhdB